### PR TITLE
Image Classification

### PR DESCRIPTION
**Pull request by srikanth-sfu**
Please navigate to tasks/computer-vision/image-classification and do 'mlflow run .'

Expect to obtain ~68% test accuracy after 200 epochs of training.

### CODE REVIEW

**Comment by slavakurilyak** [comment]
@srikanth-sfu Thanks for creating this model! Please update the `README.md` to include dataset download instructions.

**Comment by srikanth-sfu** [comment]
Dataset download happens automatically -- you don't have to download it manually, and just need a few hundred megabytes.

**Comment by mlaradji** [comment]
Hi @srikanth-sfu. Thank you for your pull request. 

I like the large number of passable parameters in `MLproject` :+1:. I ran a test run with `mlflow run . -P epochs=10`, which succeeded without error :100::
```
Train Epoch: 10 [0/50000 (0%)]	Loss: 1.208239
Train Epoch: 10 [12800/50000 (26%)]	Loss: 1.373618
Train Epoch: 10 [25600/50000 (51%)]	Loss: 1.378196
Train Epoch: 10 [38400/50000 (77%)]	Loss: 1.471107
Saving current best model....

Test set: Average loss: 1.7339, Accuracy: 5279/10000 (53%)

Training Done. Accuracy = 52.790000
Best model saved in models/
```
That's 53% accuracy with only 10 epochs! Changing the hyperparameters might improve the accuracy with 200 epochs, but 68% is good.

I have a few questions:
1. How does one set the parameter `enable-cuda` to `False`? Namely,
will `[True, 'True', 'true', 1]` all map to `True` and everything else maps to `False`?
2. After 200 epochs, does the improvement in accuracy mostly level off?
3. How long did it take to do 200 epochs on your setup?

**Comment by srikanth-sfu** [comment]
Hi @mlaradji,

1. Currently, it works only for True/False -- true/1 doesn't work as of now (Note True/'True' are equivalent here.).

2. Yes. Training loss, which is what we need to track, stabilizes towards the end (Note a caveat where there is a brief increase in training loss for a couple of epochs and fallback again to the lowest of its values soon) -- accuracy behaves in an analogous manner.

3. On my GTX 1080 Ti GPU, it took me 1.5 hours to train for 200 epochs.

**Comment by slavakurilyak** [comment]
@srikanth-sfu Please delete your `README_srikanth.md` file since you already have `tasks/computer-vision/image-classification/9945345b-9c32-45ba-a079-b9f7d6a2f515/README.md`

**Comment by srikanth-sfu** [comment]
Done!

**Comment by slavakurilyak** [comment]
@srikanth-sfu Thanks for making these modifications!

@mlaradji This pull request is ready to be merged.
